configuration/persistence.md

```diff
@@ -46,10 +46,14 @@ This section allows you to define strategies and to declare a set of default str
 ```java
 Strategies {
   <strategyName1> : "cronexpression1>"
+
+  default = everyChange
   ...
 }
 ```
 
+If no strategy is specified in an `itemlist` as described in the Items section below, the `default` strategy will be applied. The `default` parameter is optional and may be omitted.
```

Comment:
1. Please break down to one line per sentence.
2. This may be a bit confusing for beginners. 
On one side the default parameter is used when no other is given but in the next sentence this is explained as optional.
Maybe it would be good to make clear that `default` has to be set when one doesn't want to use explicit strategies only.

Its maybe a really small thing, but i stocked while reading through it and wanted to mention it.

Reply:
I agree completely.  How about this...

The `default` parameter assigns a strategy to be used if one is not specified in the `Items` section below.
The `default` parameter may be omitted from the `Strategies` section, but if and only if a strategy is provided in each line of the `Items` section.

Replace the text in the Items section with the following:
Note: if the `strategy` portion of the itemlist is omitted, the `default` strategy specified in the `Strategies` section will be applied.

Reply:
Sounds good. 👍